src/main/java/seedu/address/logic/commands/ListTuitionClassCommand.java

```diff
@@ -9,9 +9,9 @@
 /**
  * Lists all persons in the address book to the user.
  */
-public class ListTuitionClassCommand extends Command {
+public class ListTuitionClassCommand extends ListCommand {
 
-    public static final String COMMAND_WORD = "list_c";
+    //public static final String COMMAND_WORD = "list_c";
 
     // the full string that calls list class which is used by UI
     // please update after changing list_c to list class/list c
```

Comment:
Similar to ListStudentCommand, will need to change line19 to public static final String COMMAND_LIST_CLASS_STRING = "list class";